index.js

```diff
@@ -16,3 +17,51 @@ closeBtn.addEventListener('click', () => {
   nav.style.display = 'block';
   border.style.display = 'none';
 });
+
+popupClose.addEventListener('click', () => {
+  document.querySelector('.popup-window').style.display = 'none';
+  document.querySelector('body').style.overflow = 'auto';
+});
+
+document.querySelectorAll('.work_main button').forEach((btn) => {
+  btn.addEventListener('click', () => {
+    const data = {};
+
+    if (btn.parentElement.parentElement.querySelector('img')) {
+      data.image = btn.parentElement.parentElement
+        .querySelector('img')
+        .getAttribute('src');
+      document
+        .querySelector('.card-contents img')
+        .setAttribute('src', data.image);
+    } else {
+      data.image = window
+        .getComputedStyle(btn.parentElement)
+        .getPropertyValue('background-image')
+        .split('/');
+      document
+        .querySelector('.card-contents img')
+        .setAttribute(
+          'src',
+          `./workImg/${data.image[data.image.length - 1].replace('")', '')}`,
+        );
+    }
+    if (btn.parentElement.querySelector('h4')) {
+      data.title = btn.parentElement.querySelector('h4').innerText;
+      document.querySelector('.popup-header h2').innerText = data.title;
+    }
+    if (btn.parentElement.querySelector('p')) {
+      data.description = btn.parentElement.querySelector('p').innerText;
+      document.querySelector('.card-contents p').innerText = data.description;
+    }
+    if (btn.parentElement.querySelector('ul')) {
+      data.list = btn.parentElement.querySelector('ul').innerText;
+      document.querySelector('.popup-card ul').innerHTML = '';
+      data.list.split('\n').filter((m) => m !== undefined).forEach((m) => {
+        document.querySelector('.popup-card ul').innerHTML = `${document.querySelector('.popup-card ul').innerHTML}<li>${m}</li>`;
+      });
+    }
+    document.querySelector('.popup-window').style.display = 'flex';
+    document.querySelector('body').style.overflow = 'hidden';
+  });
+});
```

Comment:
- [ ]  It may be a good idea to fix your JavaScript file. Following the requirements, you should implement an array that stores objects which store the information of your projects. Example:
```javascript
const projects = [
{
    name: "Project 1",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image1.jpg",
    technologies: ["Phaser3", "HTML", "Jest"],
    link: "link_to_live_version1.com",
},
{
    name: "Project 2",
    description: "A Phaser2 plataformer game built as part of the Microverse.",
    image: "link_to_image2.jpg",
    technologies: ["Phaser3", "JavaScript", "CSS"],
    link: "link_to_live_version2.com",
}
]
```
And use them to create your project cards and send those to your HTML(You could do it with [Resource 1](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML), [Resource 2](https://www.w3schools.com/jsref/met_node_appendchild.asp)).
It means that already created HTML code for the projects list should be removed and instead it should be added via JavaScript

 After that, you can reuse this info to populate your popup window. It will allow you to create projects cards dynamically. Don't forget that your function for creating those cards in JavaScript needs to run when the page loads. 

Some resources to do it:
- [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)
- [onload](https://developer.mozilla.org/es/docs/Web/API/GlobalEventHandlers/onload)